src/SearchConfiguration.php

```diff
@@ -11,6 +11,10 @@
  */
 class SearchConfiguration
 {
+    private bool|string $wildcardAllTokens = false;
```

Comment:
In my opinion, this can also be added (with the defaults) to the constructor. I know it's getting big and I would probably use a fluent interface as well if I had to redo it. But it seems still better to me than introducing a second syntax?

Reply:
I’m okay with that even though I dislike constructors this big as well.